### ISSUE TITLE
USER ISSUE: Mark loan as failed crash

### ISSUE THREAD

**Issue by SirPewPew** [StrangeLoopGames/EcoIssues#24926]
\
**User Id: (slg234052)**
**Version:** 0.10.2.1 beta release-647

**Steps to Reproduce:**
Create loan at contract board with default clause and conditions
Loan accepted 
Loan becomes overdueForgive the debt through currency debt menu
Mark contract as failed in contract manager
Crash

**Expected behavior:**
Mark contract as failed and remove references to it

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Economy.Contracts.ContractClauseFinanceBase.Fill(FinanceAction action)
   at Eco.Gameplay.Economy.Contracts.ContractClauseFinanceBase.TryFail(GameActionPack pack, User contractor, User client)
   at Eco.Gameplay.Economy.Contracts.Contract.<MarkJobFailed>b__109_0(ContractClause clause, GameActionPack pack)
   at Eco.Gameplay.Economy.Contracts.Contract.ConfigureAndTryPerform(WorkableAction action, Action`2 checkClause, User citizen, Boolean notify, Boolean forced)
   at Eco.Gameplay.Economy.Contracts.Contract.MarkJobFailed(User user)
   at Eco.Gameplay.Economy.Contracts.Cont</size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
Unable to test without modsxp benefits, nid toolbox and a custom mod only effecting specialty stats and level gain

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/15256111/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/15256112/Player-prev.log)

**Comment by D3nnis3n** [comment]
Please provide the full server log containing the crash.

**Comment by D3nnis3n** [comment]
This was fixed in 11.1.
